R/harvest_control_rules.R

```diff
@@ -1,12 +1,8 @@
-npfmc_tier3a_F <- function(ssb, fmax){
-    lrp <- 0.05
-    urp <- 1.0
-    F <- 0.0
-    if(ssb >= urp){
-        F <- fmax
-    }else if(lrp < ssb && ssb < urp){
-        F <- fmax * (ssb-lrp)/(urp-lrp)
-    }
+npfmc_tier3_F <- function(ssb, b40, f40, alpha){
```

Comment:
Let's add Roxygen docs to this at this point. I was initially passing `ssb/b40` to the function directly rather than as separate parameters, but it doesn't matter. But I'll get confused as to what the input is supposed to be soon...